Assets/Mirror/Runtime/NetworkClient.cs

```diff
@@ -98,6 +98,35 @@ public static class NetworkClient
         // scene loading
         public static bool isLoadingScene;
 
+        [RuntimeInitializeOnLoadMethod]
```

Comment:
so runtimeinitonload is called when entering play mode, even with domain reload disabled?

Reply:
Yes, thats the unity recommended way of resetting statics without domain reload

Reply:
Added comment above it

Reply:
This was merged into Shutdown, which was renamed to ResetStatics per discord convo